panda-utilities/src/test/java/org/panda_lang/utilities/commons/ValidationUtilsTest.java

```diff
@@ -0,0 +1,22 @@
+package org.panda_lang.utilities.commons;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class ValidationUtilsTest {
+    @Test
+    void notNull__null_argument_throws_exception() {
+        Assertions.assertThrows(
+                IllegalArgumentException.class,
+                () -> ValidationUtils.notNull(null)
+        );
+    }
+
+    @Test
+    void notNull__non_null_argument_returns_argument() {
+        final String value = "testValue";
```

Comment:
We don't really use final notation for local variables. I also think that this case might be shortened by moving value and actual value directly to assertEquals statement - exactly as in the method above